docs/gettingstarted/index.rst

```diff
@@ -132,12 +129,8 @@ Getting started with Flocker
 
          Install the Flocker client on your Linux machine:
 
-         .. version-code-block:: console
-
-            you@laptop:~$ sudo apt-get update && sudo apt-get install -y gcc python2.7 python-virtualenv python2.7-dev && \
-              virtualenv flocker-tutorial && \
-              flocker-tutorial/bin/pip install --upgrade pip && \
-              flocker-tutorial/bin/pip install --quiet https://clusterhq-archive.s3.amazonaws.com/python/Flocker-|latest-installable|-py2-none-any.whl && source flocker-tutorial/bin/activate
+         .. task:: install_cli ubuntu-14.04
```

Comment:
If we want to go down that route, we should do it consistently through the documentation (so this can be a followup). On the other hand, I find reading the one with all the `&&` more difficult, and harder to see what is going on.